frontend/packages/multicluster-sdk/src/api/useFleetSearchPoll.ts

```diff
@@ -254,6 +298,39 @@ export const useFleetSearchPoll: UseFleetSearchPoll = (watchOptions, advancedSea
             setIfDefined(resource, 'status.phase', item.phase)
             setIfDefined(resource, 'spec.vmiName', item.vmiName)
             break
+          case 'VirtualMachineSnapshot.snapshot.kubevirt.io': {
+            const conditions: any = []
+            setIfDefined(conditions, `[${conditions.length}]`, item.ready, {
+              type: 'Ready',
+              status: item.ready,
+            })
+            if (conditions.length) {
+              setIfDefined(resource, 'status.conditions', conditions)
+            }
+            setIfDefined(resource, 'status.phase', item.phase)
+            if (item.indications && Array.isArray(item.indications)) {
+              setIfDefined(resource, 'status.indications', item.indications)
+            }
+            setIfDefined(resource, 'spec.source.kind', item.sourceKind)
+            setIfDefined(resource, 'spec.source.name', item.sourceName)
+            setIfDefined(resource, 'status.readyToUse', item.readyToUse)
+            break
+          }
+          case 'VirtualMachineRestore.snapshot.kubevirt.io': {
+            const conditions: any = []
+            setIfDefined(conditions, `[${conditions.length}]`, item.ready, {
+              type: 'Ready',
+              status: item.ready,
+            })
+            if (conditions.length) {
+              setIfDefined(resource, 'status.conditions', conditions)
+            }
```

Comment:
Same comment as other spot with only 1 status condition - don't need to construct `conditions` separately.